molecule/default/requirements.yml

```diff
@@ -0,0 +1,6 @@
+---
+# from galaxy
+- src: geerlingguy.repo-epel
```

Comment:
We've not got this in the dependencies for the role. Do you know what we pull from EPEL?